crates/mempool/src/lib.rs

```diff
@@ -0,0 +1,274 @@
+// Copyright 2024 PRAGMA
```

Comment:
```suggestion
// Copyright 2025 PRAGMA
```

Reply:
Note: this year doesn't necessarily need to reflect the current year, but only the year from which this file began being licensed under those terms. So if the work started in 2024, this can say 2024 👍